bibliopixel/animation/sequence.py

```diff
@@ -11,20 +11,21 @@ def make_animation(a):
                 desc = a
             else:
                 desc = {'animation': a[0], 'run': a[1]}
-            desc = aliases.resolve_aliases(desc)
+            desc = aliases.resolve(desc)
+            # desc['run']['threaded'] = True
```

Comment:
I mean the comment on line 15.

I meant "either delete the code entirely, or put it in an if statement, don't comment it out"!